jovian/utils/records.py

```diff
@@ -59,11 +60,14 @@ def log_record(record_type, data=None, verbose=True, **data_args):
     tracking_slug = res['tracking']['trackingSlug']
     # Save to data block
     _data_blocks.append((tracking_slug, record_type, data))
+
+    if notify:
```

Comment:
how about doing this from server directly? You can pass notify param along with log_* calls

Reply:
Let's have discussion and do this in a next PR. Currently merging this for updated logo

Reply:
you can create a separate PR for updated logo

Reply:
Done